src/operators/index.ts

```diff
@@ -28,7 +28,7 @@ export { distinctUntilKeyChanged } from '../internal/operators/distinctUntilKeyC
 export { elementAt } from '../internal/operators/elementAt';
 export { endWith } from '../internal/operators/endWith';
 export { every } from '../internal/operators/every';
-export { exhaust } from '../internal/operators/exhaust';
+export { exhaust, exhaustAll } from '../internal/operators/exhaust';
```

Comment:
Hi. Does it make sense to rename `exhaust.ts` file to `exhaustAll.ts`? Would such change break anything?

Reply:
I believe not as it lives in the internal folder. Good point!
After figuring out how to fix the breaking build ill ask for it.